contrib/babelfishpg_tsql/src/linked_servers.c

```diff
@@ -166,6 +165,25 @@ linked_server_err_handler(LinkedServerProcess lsproc, int severity, int db_error
 
 	appendStringInfo(&buf, "Level: %i", severity);
 
+	return buf;
+}
+
+/*
+ * Handle any error encountered in TDS client library itself
+ */
+static int
+linked_server_err_handler(LinkedServerProcess lsproc, int severity, int db_error, int os_error, char *db_err_str, char *os_err_str)
+{
+	StringInfoData buf;
+
+	buf = construct_err_string(severity, db_error, os_error, db_err_str, os_err_str);
+
+	if (db_error == SYBETIME)
+	{
+		isQueryTimeout = true;
```

Comment:
Add a comment as to why we are not throwing the error here

Reply:
done

Reply:
done